platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBenchJPDB.java

```diff
@@ -47,10 +44,27 @@ protected VirtualMap<BenchmarkKey, BenchmarkValue> createEmptyMap() {
         final JasperDbBuilder<BenchmarkKey, BenchmarkValue> diskDbBuilder = new JasperDbBuilder<>();
         diskDbBuilder
                 .virtualLeafRecordSerializer(virtualLeafRecordSerializer)
-                .virtualInternalRecordSerializer(new VirtualInternalRecordSerializer())
+                .virtualInternalRecordSerializer(new PathHashRecordSerializer())
                 .keySerializer(new BenchmarkKeySerializer())
                 .storageDir(getTestDir().resolve("jasperdb"))
                 .preferDiskBasedIndexes(false);
         return new VirtualMap<>(LABEL, diskDbBuilder);
     }
+
+    public static void main(final String[] args) throws Exception {
```

Comment:
just a convenience method to run the benchmark in IDE

Reply:
Please remove. This way of running benchmarks should not and will not be maintained.
(You can stash and maintain your local copy if you find it useful).

Reply:
Removed